app/styles/app.scss

```diff
@@ -211,11 +211,19 @@ body {
   border: none;
 }
 
-@media (max-width: 760px) {
-  .navbar .navbar-brand {
-      justify-content: center;
-      margin: auto;
-  }
+.border-none {
+  border: none!important;
+}
+.img-avatar {
+  width: 30px;
+}
+.cus-header {
```

Comment:
cus?

Reply:
oooh custom - might be better with a different name, either written out or changed both work!

Reply:
Updated to custom-header although this class will be removed later on when building out the landing page further.